package.json

```diff
@@ -4,11 +4,17 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "start": "npx webpack-dev-server --mode development --open --hot",
     "build": "npx webpack --mode production",
     "calc": "npx ts-node src/lesson2",
     "lint": "npx eslint --ext .js,.jsx,.ts,.tsx --fix ./",
+    "loki:update": "npm run build-storybook && npx loki update --requireReference --reactUri file:./storybook-static",
+    "loki:approve": "npx loki approve",
+    "test:loki": "npm run build-storybook && npx loki --requireReference --reactUri file:./storybook-static",
     "test": "npx jest",
-    "check": "npm test && npm run lint"
```

Comment:
Было бы здорово добавить скрипт для проверки типов перед тестами - сейчас чтобы проверить типы приходится запускать для каждого файла `npx tsc ./src/Typesript/easy1.ts` ...

Reply:
И еще момент, в tsconfig.json у вас нет "outDir": "./build" (например), и при этом стоит allowJs: true
Если запустить `npx tsc`  у меня появляются ошибки типа
_error TS5055: Cannot write file '/home/try/dev/react-js-tutorial/build/src/components/Cell.js' because it would overwrite input file._
js файлы создаются и все вроде бы норм, но ошибки типизации у меня при этом не показываются, а они есть.